packages/x-components/src/__tests__/utils.ts

```diff
@@ -182,21 +184,23 @@ function mergeStates<State extends Dictionary>(
 }
 
 /**
- * Makes a clean install of the {@link XPlugin} into the passed Vue object.
+ * Makes a clean installation of the {@link XPlugin} into the passed Vue object.
  * This also resets the bus, and all the hardcoded dependencies of the XPlugin.
  *
  * @param options - The options for installing the {@link XPlugin}. The
  * {@link XComponentsAdapterDummy}  is added by default.
  * @param localVue - A clone of the Vue constructor to isolate tests.
  * If not provided, one will be created.
+ * @param bus - Optional bus to be used by the XPlugin.
  * @returns An array containing the `xPlugin` singleton and the `localVue` and objects.
  */
 export function installNewXPlugin(
   options: Partial<XPluginOptions> = {},
-  localVue: typeof Vue = createLocalVue()
+  localVue: typeof Vue = createLocalVue(),
+  bus?: XPriorityBus<XEventsTypes, WireMetadata>
 ): [XPlugin, typeof Vue] {
   XPlugin.resetInstance();
-  const xPlugin = new XPlugin(new XDummyBus());
+  const xPlugin = new XPlugin(bus ?? new XDummyBus());
```

Comment:
Bypass for this PR; pending to close https://github.com/empathyco/x/pull/1447 before